core/server/api/v2/utils/index.js

```diff
@@ -9,5 +9,10 @@ module.exports = {
 
     get validators() {
         return require('./validators');
+    },
+
+    isContentAPI: (frame) => {
```

Comment:
Looks good, would improve the readability a bit... don't have a recepie for it but maybe:
```
    isContentAPI: (frame) => {
        return !!(
            Object.keys(frame.options.context).length === 0 ||
            (!frame.options.context.user && frame.options.context.api_key_id) ||
            frame.options.context.public
        );
    }
```
what do you think @rishabhgrg ?

Reply:
Looks good, I just updated with something similar, but `||` on next line. This is better, lemme update :)